.editorconfig

```diff
@@ -20,8 +17,5 @@ insert_final_newline = false
 [Makefile]
 indent_style = tab
 
-[*.{js,json,coffee,less,sass,scss,css,rb}]
-indent_size = 2
```

Comment:
I thought the reason for having it is that I might have set my default indent size to 4 in my editor, and this ensures that for this codebase it gets set to 2.